spec/features/navigation_spec.rb

```diff
@@ -74,6 +99,18 @@
       visit '/admin'
       expect(page).to have_content("The page you were looking for doesn't exist.")
     end
+    it "I can see the link to my profile page and a link to logout" do
+      user = create(:merchant_user)
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
+      visit "/"
+
+      expect(page).to have_link("Profile")
+      expect(page).to have_link("Logout")
+      expect(page).to have_no_content("Login") 
+      expect(page).to have_no_content("Register")
+      expect(page).to have_link("Dashboard") 
```

Comment:
If you change this line to `expect(page).to have_link("Dashboard", href: "/merchant")` it will test that the "Dashboard" link points to /merchant. This would be good since the dashboard link changes depending on the user's role.